### ISSUE TITLE
2 problem about using watchme

### ISSUE THREAD

**Issue by wai2000123** [youtube/yt-watchme#36]
First of all, I am already post the live stream on youtube successfully. Below is 2 problems I have met.
- The frame per second is too low, I found it from my debug log. I found that the the fps is less than 3, it makes my streaming not smooth and youtube warned me to increase the bit rate.
- The stream video do not have any sound, I have thought that is because of the fps too low but seems not really it is from watching the issues that other user recommended.

Sorry for my bad English, but I need a little help. Lets us discuss here?

**Comment by mohit-sharma-87** [comment]
@wai2000123 : Hi , I am unsuccessful to run watch-me project . Will it be possible to share your code base with me .

**Comment by sunshinetpu** [comment]
@mohit-sharma-87 Have you successfully run the project?

**Comment by mohit-sharma-87** [comment]
Yes , i was able to run it successfully but build was not stable .

**Comment by nileshgangadhar** [comment]
Hi i am unsuccessfull to run watch me project. i got Exception like that 
java.lang.UnsatisfiedLinkError: No implementation found for boolean com.google.android.apps.watchme.Ffmpeg.init(int, int, int, java.lang.String) (tried Java_com_google_android_apps_watchme_Ffmpeg_init and Java_com_google_android_apps_watchme_Ffmpeg_init__IIILjava_lang_String_2)
how to resolve this issue.Plz Help me

**Comment by vranacse** [comment]
need working demo app with both video and audio working ! contact : vranacse@gmail.com

**Comment by sourav-dev** [comment]
@vranacse Have you successfully run the project? Are you able to stream?
